src/stravalib/tests/integration/test_client.py

```diff
@@ -295,6 +316,12 @@ def test_get_activity_streams_series_type_unofficial(mock_strava_api, client):
             None,
         ),
         ({"hide_from_home": False}, {"hide_from_home": "0"}, None, None),
+        (
+            {"name": "My awesome activity"},
```

Comment:
The tiny decline is cod coverage should be addressed by this additional test. 
🤞 
<img width="426" alt="Screenshot 2024-03-18 at 7 42 49 PM" src="https://github.com/stravalib/stravalib/assets/7649194/c467aca9-0bcb-49db-af62-badc62f29fad">